### PR TITLE
Allow to add optional tags in new vacancy form

### LINKED ISSUE #17: Allow to add optional tags in new vacancy form
## What
We want to use tags in the future for filtering. For this we need to add ability to add it now.

**IMPORTANT**: this issue is just about form and adding tags, you don't need to show it

## How

Just add new field in to form (tags), after that [update mapper](https://github.com/davydovanton/rubyjobs.dev/blob/master/lib/vacancies/mappers/vacancy.rb) to convert string to array of strings and that's all.

### PR DESCRIPTION
**Pull request by mrlewap**
Closes #17

### CODE REVIEW

File: lib/vacancies/mappers/vacancy.rb [hunk 1]
```diff
@@ -22,7 +22,9 @@ def call(payload) # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
             salary_currency: payload[:salary][:currency],
             salary_unit: payload[:salary][:unit],
 
-            archived_in_weeks: payload[:archived_in_weeks].to_i
+            archived_in_weeks: payload[:archived_in_weeks].to_i,
+
+            tags: payload[:tags].split
```
Comment (davydovanton) [inline lib/vacancies/mappers/vacancy.rb]: I think we need to split it only by `,` char, WDYT?

Comment (davydovanton) [reply lib/vacancies/mappers/vacancy.rb]: and also, could you downcase the string before?

**Comment by mrlewap** [comment]
ok, will do requested changes!

**Comment by mrlewap** [comment]
@davydovanton done

**Pull request merged**